tests/wpt/metadata/XMLHttpRequest/data-uri.htm.ini

```diff
@@ -1,6 +1,5 @@
 [data-uri.htm]
   type: testharness
-
```

Comment:
This is the result of update-wpt, others doing the same command will have the same result, should I re-add the deleted line ?